renku/ui/cli/workflow.py

```diff
@@ -622,6 +622,14 @@
 This will allow you to navigate between workflow execution and see details
 by pressing the <Enter> key.
 
+If you prefer to elaborate the output graph further, or if you wish to export
+it for any reason, you can use the ``--format`` option to specify an output
+format.
+
+.. code-block:: console
+
+   $ renku workflow visualize --format dot <path>
+
```

Comment:
Maybe write a sentence that this would print dot format, which could be accepted by the `dot` command